ipn/localapi/localapi.go

```diff
@@ -125,8 +125,8 @@ var (
 	metrics   = map[string]*clientmetric.Metric{}
 )
 
-func NewHandler(b *ipnlocal.LocalBackend, logf logger.Logf, logID logid.PublicID) *Handler {
-	return &Handler{b: b, logf: logf, backendLogID: logID}
+func NewHandler(b *ipnlocal.LocalBackend, logf logger.Logf, netMon *netmon.Monitor, logID logid.PublicID) *Handler {
```

Comment:
docs on nil-ness.

(sorry, no docs previously anyway)

I assume b is required, maybe logf too?, and netMon apparently not?